### ISSUE TITLE
Cannot POST /  - statusCode": 404

### ISSUE THREAD

**Issue by proutek** [nestjs/graphql#397]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Since 6.4.3 I'm getting response to all my GQL requests
My path is set as **path: '/',**
``` json
{
    "statusCode": 404,
    "error": "Not Found",
    "message": "Cannot POST /"
}
```
When I revert this change in graph.module.ts, it's working again.

``` javascript
        // Commented out - DOESN'T WORK
        // const path = useGlobalPrefix
        //     ? normalize_route_path_util_1.normalizeRoutePath(prefix) + gqlOptionsPath
        //     : gqlOptionsPath;
        const path = useGlobalPrefix
            ? prefix + this.options.path
            : this.options.path;
```

The cause is commit https://github.com/nestjs/graphql/commit/95872f115cb0b443d5c25069341780cb4d328980

## Expected behavior
GQL request are working

## Environment

<pre><code>
Nest version: 6.4.3 - 6.5.0

 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by proutek** [comment]
Possible fix

``` javascript
function stripEndingSlash(text) {
  if (!text) {
    return text;
  }
  return text[text.length - 1] === '/' && text.length > 1
    ? text.slice(0, text.length - 1)
    : text;
}
```

**Comment by kamilmysliwiec** [comment]
Thanks for reporting :) Fixed in 6.5.1
